### PR TITLE
Generate and enforce a single manifest for deploying frr-k8s

### PR DESCRIPTION
**Pull request by fedepaol**

### CODE REVIEW

File: config/all-in-one/frr-k8s.yaml [hunk 1]
```diff
@@ -0,0 +1,800 @@
...
+metadata:
+  labels:
+    app.kubernetes.io/component: kube-rbac-proxy
+    app.kubernetes.io/created-by: frr-k8s
+    app.kubernetes.io/instance: frr-k8s-metrics-service
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: service
+    app.kubernetes.io/part-of: frr-k8s
+    control-plane: frr-k8s
+  name: frr-k8s-frr-k8s-metrics-service
```
[the remark this answers is not in the document]
Comment (fedepaol) [reply config/all-in-one/frr-k8s.yaml:556]: yep I thought I caught them all but this went through. Fixing

File: config/all-in-one/frr-k8s.yaml [hunk 2]
```diff
@@ -0,0 +1,800 @@
...
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app.kubernetes.io/component: frr-k8s
+    app.kubernetes.io/created-by: frr-k8s
+    app.kubernetes.io/instance: frr-k8s
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: deployment
```
[the remark this answers is not in the document]
Comment (fedepaol) [reply config/all-in-one/frr-k8s.yaml:575]: probably frr-k8s or something along that line

[the remark this answers is not in the document]
Comment (fedepaol) [reply config/all-in-one/frr-k8s.yaml]: changed

**Comment by liornoy** [comment]
lgtm

**Comment by liornoy** [comment]
notice we need to rebase this with main

**Comment by fedepaol** [comment]
> notice we need to rebase this with main

With merge queues it's not needed

**Pull request merged**